### PR TITLE
Optimize parametrization

### PR DESCRIPTION
**Pull request by bliemli**

### CODE REVIEW

File: manifests/role/ansible_master.pp [hunk 1]
```diff
@@ -36,20 +36,22 @@
   $host_groups,
   $playbooks_source = 'https://github.com/openshift/openshift-ansible.git',
   $playbooks_version = 'master',
+  $ansible_inventory = '/etc/ansible/hosts',
+  $packages = [
```
Comment (hansmi) [inline manifests/role/ansible_master.pp:40]: Adding more packages requires duplicating the existing values, and list updates become more difficult. I presume you want to define a list on your own. You can do that by adding a new boolean parameter named like `$install_required_packages`. That way external code can install the desired packages.

File: manifests/role/ansible_master.pp [hunk 2]
```diff
@@ -36,20 +36,22 @@
...
+    'ansible',
+    'git',
+    'jq',
+    'pyOpenSSL',
+    'wget',
+  ],
 ) {
   validate_hash($host_groups)
```
Comment (hansmi) [inline manifests/role/ansible_master.pp:48]: Missing `validate_absolute_path($ansible_inventory)` (feel free to add validations for the other parameters in a separate commit).

**Review by hansmi — CHANGES_REQUESTED** [review]
The commit messages should start with a capitalized letter and contain a body (see, for example, https://chris.beams.io/posts/git-commit/).

**Comment by bliemli** [comment]
Just reviewed with Lorenz. We are closing this PR and will open a new one.

**Comment by bliemli** [comment]
Just reviewed with Lorenz. We are closing this PR and will open a new one.

**Pull request closed without merging**